sdk/tables/data-tables/tests.yml

```diff
@@ -1,12 +1,13 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/data-tables"
-    ResourceServiceDirectory: tables
-    TestSamples: false
-    EnvVars:
-      AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-      AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/data-tables"
+      ServiceDirectory: tables
+      MatrixFilters:
```

Comment:
Should we also exclude samples? Or does this pipeline now support samples?

Reply:
For all pipelines including samples that didn't before, I have verified they pass! Ideally we run as many samples as we can.